srcpkgs/loudgain/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'loudgain'
+pkgname=loudgain
+version=0.6.8
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="libebur128-devel ffmpeg-devel taglib-devel pkg-config"
```

Comment:
`pkg-config` should only be in `hostmakedepends`

Reply:
Thanks , don't know how I missed that , just fixed it !